xwiki-rendering-wikimodel/src/main/java/org/xwiki/rendering/wikimodel/tex/TexSerializer.java

```diff
@@ -465,7 +467,7 @@ protected InputStream getImageInput(String ref) throws IOException
                 WGet wget = new WGet();
                 wget.fetchURL(url, fos);
             } catch (Exception e) {
-                e.printStackTrace();
+                LOGGER.error("Error while getting image input",e);;
```

Comment:
This class is an old class used by wikimodel that we integrated and don't use. AFAICS it's not finished work and uses System.out/err in several places. I think we might be better off simply removing this class. Just need to check that we really don't use it (we have a latex Renderer AFAIR).

Small note: you forgot a space after the commas.

Also, when writing debug logs you need to provide the max relevant info so that it can be debugged easily. For example you forgot to add the URL or some kind of image reference to identify the image input.

Thanks!